modules/installation/pages/install-uyuni.adoc

```diff
@@ -0,0 +1,79 @@
+[[install-uyuni]]
+= Install Uyuni Server with openSUSE
+
+{productname} Server can be installed on openSUSE.
+
+[NOTE]
+====
+This table specifies the minimum requirements.
+====
+
+In this table, replace ``version`` with the version of the product you are using.
+
+[cols="1,1", options="header"]
+.Hardware and Software Requirements
+|===
+| Requirement            | Description
+| openSUSE Leap 15.1     | Clean installation, up-to-date
+| Multi-core CPU         | x86_64
+| 8 GB RAM *) 	         | Sufficient for testing purposes
+| 100 GB Disk Space **)  | Disk space depends on your channel requirements
+|===
+
+*) 32 GB of RAM recommended for a production environment (16 GB for a base installation).
+
+**) 50 GB per SUSE or openSUSE product and 360 GB per Red Hat product.
+
+
+////
+[NOTE]
+====
+Storage size values are the absolute minimum—only suitable for a small test or demo installation.
+Especially [path]``/var/spacewalk/`` may quickly need more space.
+Also consider to create a separate partition for [path]``/srv`` where Kiwi images are stored.
+====
+////
+
+////
+[cols="1,1,1", options="header"]
+|===
+| VirtIO Storage Disks | Name      | Sizing
+| VirtIO Disk 2        | spacewalk | 101{nbsp}GB
+| VirtIO Disk 3        | pgsql     | 50{nbsp}GB
+| VirtIO Disk 4        | swap      | 4{nbsp}GB
+|===
+////
+
+== Install openSUSE Leap 15.1.
+If you want to re-use an existing openSUSE Leap 15.1 installation make sure that all available package updates are installed.
```

Comment:
The instance has to be brand new and fully updated. You just said it above :-)

Reply:
ok, I changed it for clarity.